Userland/Libraries/LibWeb/MimeSniff/MimeType.cpp

```diff
@@ -330,40 +330,11 @@ bool MimeType::is_scriptable() const
 // https://mimesniff.spec.whatwg.org/#javascript-mime-type
 bool MimeType::is_javascript() const
 {
-    // A JavaScript MIME type is any MIME type whose essence is one of the following:
-    //    - application/ecmascript
-    //    - application/javascript
-    //    - application/x-ecmascript
-    //    - application/x-javascript
-    //    - text/ecmascript
-    //    - text/javascript
-    //    - text/javascript1.0
-    //    - text/javascript1.1
-    //    - text/javascript1.2
-    //    - text/javascript1.3
-    //    - text/javascript1.4
-    //    - text/javascript1.5
-    //    - text/jscript
-    //    - text/livescript
-    //    - text/x-ecmascript
-    //    - text/x-javascript
-    return essence().is_one_of(
-        "application/ecmascript"sv,
-        "application/javascript"sv,
-        "application/x-ecmascript"sv,
-        "application/x-javascript"sv,
-        "text/ecmascript"sv,
-        "text/javascript"sv,
-        "text/javascript1.0"sv,
-        "text/javascript1.1"sv,
-        "text/javascript1.2"sv,
-        "text/javascript1.3"sv,
-        "text/javascript1.4"sv,
-        "text/javascript1.5"sv,
-        "text/jscript"sv,
-        "text/livescript"sv,
-        "text/x-ecmascript"sv,
-        "text/x-javascript"sv);
+    for (const auto& javascript_essence : javascript_mime_type_essence_strings) {
+        if (essence() == javascript_essence)
+            return true;
+    }
+    return false;
```

Comment:
You could use `contains_slow` here:
```suggestion
    return javascript_mime_type_essence_strings.contains_slow(essence());
```

Reply:
Good idea, thanks!